docs/pages/machine-id/access-guides/ansible.mdx

```diff
@@ -31,7 +37,77 @@ You will need the following tools to use Teleport with Ansible.
 
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/2. Configure Ansible
+## Step 1/4. Configure RBAC
+
+First, Teleport must be configured to allow the credentials produced by the bot
```

Comment:
This is the first mention of "the bot", and I think we can be more explicit as to whether this refers to `tbot` or Ansible.

This sentence also throws me off a little bit because it says that the credentials produced by the bot connect to SSH nodes. Should it instead say something like,

```markdown
First, Teleport must be configured to allow Ansible to connect to your SSH nodes. Ansible can then use the credentials produced by `tbot` to authenticate to the nodes.
```

Reply:
https://github.com/gravitational/teleport/pull/32741/commits/1001c046259cfb3a1e4bbee718f204874f4252f6